Fastlane: Add Classic Checkout card design to Block Checkout (4996)

Narek13 opened the pull request:
## Issue Description

We need Fastlane credit card placeholder to look nice on Classic and Block Checkout.
The Classic Checkout has a more modern card place holder design in Ryan flow:

<img width="1083" height="840" alt="image-20250710-170015" src="https://github.com/user-attachments/assets/8c05fbfe-a2e4-48b0-b6ab-fe8f2f37b93b" />

The Block Checkout has an older design:

<img width="1424" height="1067" alt="image-20250710-170018" src="https://github.com/user-attachments/assets/318864a9-aa63-480b-a09f-018764a3954b" />

## PR Description

This PR adjusts the styles and structure of block to match the design of Classic checkout

Narek13 committed:
Adjust styles

Narek13 committed:
Adjust component

danieldudzic reviewed (changes requested):
@Narek13 The code looks good!

There still seem to be some visual discrepancies between the checkouts:

<img width="1460" height="638" alt="Checkout_–_WooCommerce_PayPal_Payments_i_Classic_Checkout_–_WooCommerce_PayPal_Payments" src="https://github.com/user-attachments/assets/befc9f32-1f95-487b-8b3f-cd61108d3a2b" />

Narek13 commented:
Thanks @danieldudzic
I thinks the fonts are a bit different, but I was thinking the logo should stay?

However I have removed the watermark and fixed the fonts

Narek13 committed:
Fix the fonts

Narek13 committed:
Remove the watermark

danieldudzic commented:
@InpsydeNiklas Do you think we should keep the watermark/add it on the classic checkout or have it on the block checkout and not in the classic checkout?
